Clinical-Genomics/scout
# Clean variants [or dry run clean variants] and estimate freed space on disk

fix #2200
Allows to remove variants from one case or all cases thus freeing up space in the database.

Options:
![image](https://user-images.githubusercontent.com/28093618/100255923-67c24a80-2f44-11eb-9c4e-d0f12025b628.png)

Default status for the options above: 
- default values for status is: ["solved", "archived", "migrated"]
- older-than : 0 # Remove from any case, regardless of how old the analysis is
- min_rank-threshold: 5
- variants-threshold: doesn't have a default (not provided)

I haven't written tests yet because I could use some feedback over the functionality first.. Like did I miss anything or would it be better to change any default value?

**How to test**:
1. Test locally with the demo instance. Better by loading more than one case, for instance load also the other RD case and the cancer case:
- scout --demo load case scout/demo/643595.config.yaml
- scout --demo load case scout/demo/cancer.load_config.yaml

- Run the command: `scout --loglevel INFO  --demo delete variants` + options
note that the demo data has status=inactive by default, so either you change the status of the case in the web page or you provide the -status inactive param using the delete command above

1. Then I would do also a dry run on scout stage on hasta.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
